### ISSUE TITLE
can not make upgarde

### ISSUE THREAD

**Issue by Mister-XY** [openmediavault/openmediavault#1651]
Hello, I downloaded your image openmediavault_7.0-11-amd64, unfortunately it is not possible to upgrade with this image because there seem to be leftovers from the raspi on the AMD64 image.
Unfortunately I haven't found a way to remove this.

```root@NAS-Steinke:~# apt upgrade
Paketlisten werden gelesen… Fertig
Abhängigkeitsbaum wird aufgebaut… Fertig
Statusinformationen werden eingelesen… Fertig
Paketaktualisierung (Upgrade) wird berechnet… Fertig
0 aktualisiert, 0 neu installiert, 0 zu entfernen und 0 nicht aktualisiert.
2 nicht vollständig installiert oder entfernt.
Nach dieser Operation werden 0 B Plattenplatz zusätzlich benutzt.
Möchten Sie fortfahren? [J/n] j
initramfs-tools (0.142) wird eingerichtet ...
update-initramfs: deferring update (trigger activated)
linux-image-6.1.0-15-amd64 (6.1.66-1) wird eingerichtet ...
/etc/kernel/postinst.d/initramfs-tools:
update-initramfs: Generating /boot/initrd.img-6.1.0-15-amd64
raspi-firmware: missing /boot/firmware, did you forget to mount it?
run-parts: /etc/initramfs/post-update.d//z50-raspi-firmware exited with return code 1
run-parts: /etc/kernel/postinst.d/initramfs-tools exited with return code 1
dpkg: Fehler beim Bearbeiten des Paketes linux-image-6.1.0-15-amd64 (--configure):
 »installiertes post-installation-Skript des Paketes linux-image-6.1.0-15-amd64«-Unterprozess gab den Fehlerwert 1 zurück
Trigger für initramfs-tools (0.142) werden verarbeitet ...
update-initramfs: Generating /boot/initrd.img-6.1.0-15-amd64
raspi-firmware: missing /boot/firmware, did you forget to mount it?
run-parts: /etc/initramfs/post-update.d//z50-raspi-firmware exited with return code 1
dpkg: Fehler beim Bearbeiten des Paketes initramfs-tools (--configure):
 »installiertes post-installation-Skript des Paketes initramfs-tools«-Unterprozess gab den Fehlerwert 1 zurück
Fehler traten auf beim Bearbeiten von:
 linux-image-6.1.0-15-amd64
 initramfs-tools
E: Sub-process /usr/bin/dpkg returned an error code (1)

apt remove raspi-firmware
Paketlisten werden gelesen… Fertig
Abhängigkeitsbaum wird aufgebaut… Fertig
Statusinformationen werden eingelesen… Fertig
Paket »raspi-firmware« ist nicht installiert, wird also auch nicht entfernt.
0 aktualisiert, 0 neu installiert, 0 zu entfernen und 23 nicht aktualisiert.
1 nicht vollständig installiert oder entfernt.
Nach dieser Operation werden 0 B Plattenplatz zusätzlich benutzt.
initramfs-tools (0.142) wird eingerichtet ...
update-initramfs: deferring update (trigger activated)
Trigger für initramfs-tools (0.142) werden verarbeitet ...
update-initramfs: Generating /boot/initrd.img-6.1.0-13-amd64
raspi-firmware: missing /boot/firmware, did you forget to mount it?
run-parts: /etc/initramfs/post-update.d//z50-raspi-firmware exited with return code 1
dpkg: Fehler beim Bearbeiten des Paketes initramfs-tools (--configure):
 »installiertes post-installation-Skript des Paketes initramfs-tools«-Unterprozess gab den Fehlerwert 1 zurück
Fehler traten auf beim Bearbeiten von:
 initramfs-tools
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

edit: i fixed it withe the command

dpkg --purge raspi-firmware

i hope that will help other user with this problem.

**Comment by votdev** [comment]
This seems to be a bug introduced by Debian

- https://forums.debian.net/viewtopic.php?t=155245
- https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1032071

The problem seems to be fixed in live-build with https://salsa.debian.org/live-team/live-build/-/commit/8eaf20daf1cf79669975b1acfe4d6fa453eb6307. Lets see if building new ISO images with that package version will fix the issue.

**Comment by votdev** [comment]
Workarounds are mentioned here:

- https://www.thomas-krenn.com/de/wiki/Raspi-firmware:_missing_/boot/firmware,_did_you_forget_to_mount_it
- https://soltveit.org/debian-12-linux-image-6-1-0-10amd64/

**Comment by ryecoaaron** [comment]
https://forum.openmediavault.org/index.php?thread/50630-i-have-a-update-problem-with-omv7-0-11/

**Comment by votdev** [comment]
Should be fixed with the latest 7.0-15 ISO image.
